### ISSUE TITLE
[BUG] YoutubePlayerController was used after being disposed.

### ISSUE THREAD

**Issue by HiIamAlanOu** [sarbagyastha/youtube_player_flutter#115]
**Describe the bug**
When i use live broadcast function on YoutubePlayerController, and pop to the previous page.
```
Navigator.of(context).pop();
```

**Error**
```
════════ Exception caught by widgets library ═══════════════════════════════════
The following assertion was thrown while finalizing the widget tree:
A YoutubePlayerController was used after being disposed.

Once you have called dispose() on a YoutubePlayerController, it can no longer be used.
When the exception was thrown, this was the stack
#0      ChangeNotifier._debugAssertNotDisposed.<anonymous closure> 
package:flutter/…/foundation/change_notifier.dart:105
#1      ChangeNotifier._debugAssertNotDisposed 
package:flutter/…/foundation/change_notifier.dart:111
#2      ChangeNotifier.removeListener 
package:flutter/…/foundation/change_notifier.dart:166
#3      _PlayPauseButtonState.dispose 
package:youtube_player_flutter/…/widgets/play_pause_button.dart:60
#4      StatefulElement.unmount 
package:flutter/…/widgets/framework.dart:4142
...
════════════════════════════════════════════════════════════════════════════════
```

**Comment by sarbagyastha** [comment]
@HiIamAlanOu Will publish with fix tonight.

**Comment by HiIamAlanOu** [comment]
Thank u.

**Comment by sarbagyastha** [comment]
Fixed in **Version 6.0.1**
